wow/tests/test_datautils.py

```diff
@@ -16,3 +16,18 @@ def test_str_or_none_works(self):
         assert str_or_none(None) is None
         assert str_or_none('1') == '1'
         assert str_or_none(1) == '1'
+
+    def test_json_or_none_works(self):
+        sample_json_string = '[{"foo": "bar", "beep": { "boop": "blah"}}]'
+        sample_json_string_with_null = '[{"foo": "bar", "beep": null}]'
+        assert json_or_none(None) is None
+        assert json_or_none(sample_json_string) == [{
+            "foo": "bar",
+            "beep": {
+                "boop": "blah"
+            }}
+        ]
+        assert json_or_none(sample_json_string_with_null) == [{
+            "foo": "bar",
+            "beep": None
+        }]
```

Comment:
AW YEAH THE TESTS NOICE

Reply:
Oh this would be an excellent candidate for the use of [`pytest.mark.parametrize`](https://docs.pytest.org/en/6.2.x/parametrize.html) by the way.  The nice thing is that it actually will turn this one test into three separate tests with a minimum of fuss (which makes it easier to figure out what's going wrong if any of them ever break).

Reply:
you got it!